### PR TITLE
Polyglot Docs

### PR DESCRIPTION
**Pull request by natansil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,11 +53,6 @@ You can read more about it on this blog post - “[Building a High-level SDK for
 
 ## Usage
```
Comment (natansil) [inline README.md:54]: usage per language (Maven central core/future/java specifically )

Comment (natansil) [reply README.md:54]: java: https://search.maven.org/search?q=a:greyhound-java
scala future: https://search.maven.org/search?q=a:greyhound-future_2.12
ZIO https://search.maven.org/search?q=a:greyhound-core

File: README.md [hunk 2]
```diff
@@ -65,6 +62,14 @@ First let's review some basic messaging terminology:
  * Processes that subscribe to topics and process the feed of published messages are called consumers.
  * Kafka is run as a cluster comprised of one or more servers, each of which is called a broker.
 
+
+### Scala Future
+
+### Add greyhound to your build
```
Comment (natansil) [inline README.md:68]: each language/API flavor should have it's own usage

File: README.md [hunk 3]
```diff
@@ -12,510 +14,87 @@ Greyhound seeks to provide a higher-level interface to Kafka and to express rich
...
 
-Use the `strategy` config to define how the producer flushes records to Kafka:<br>
+See [examples](docs/build.md) of how to add greyhound modules to your build (Maven, Gradle, SBT, etc.)
 
-```scala
-ProduceStrategy.Sync(concurrency: Int)
-ProduceStrategy.Async(batchSize: Int, concurrency: Int)
-ProduceStrategy.Unordered(batchSize: Int, concurrency: Int)
-```
+## Testing
```
Comment (jasonbariwix) [inline README.md:94]: this Testing section should be moved to ZIO file (zio-based-api.md)

**Pull request merged**